test_closed_connection failing due to error handling issue

```
____________________________ test_closed_connection ____________________________
kdb_server = k('15i')

    def test_closed_connection(kdb_server):
...
>           assert msg == 'close'
E           AssertionError: assert 'close handle: 15' == 'close'
E             - close handle: 15
E             + close
...
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,16 +12,6 @@ matrix:
         - BITS=64
         - PY=2.7
         - NUMPY=1.16.4
-    - os: linux
-      env:
-        - BITS=32
-        - PY=3.6
-        - NUMPY=1.16.4
-    - os: linux
-      env:
-        - BITS=32
-        - PY=2.7
-        - NUMPY=1.16.4
     - os: osx
       env:
         - BITS=64
@@ -32,10 +22,6 @@ matrix:
         - PY=2.7
   allow_failures:
     - os: osx
-    - os: linux
-      env: BITS=32 PY=2.7 NUMPY=1.16.4
-    - os: linux
-      env: BITS=32 PY=3.6 NUMPY=1.16.4
 
 install:
   # We do this conditionally because it saves us some downloading if the

--- a/.travis.yml
+++ b/.travis.yml
@@ -32,13 +32,14 @@ matrix:
         - PY=2.7
   allow_failures:
     - os: osx
+    - env: BITS=32
 
 install:
   # We do this conditionally because it saves us some downloading if the
   # version is the same.
   - if [[ "$PY" == "2.7" ]]; then mc=Miniconda2; else mc=Miniconda3; fi
   - if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then os=Linux; else os=MacOSX; fi
-  - if [[ "${BITS}" = "32" ]]; then bt="x86"; else bt="x86_64"; fi
+  - if [[ "${BITS}" = "32" ]]; then bt="x86"; export CONDA_FORCE_32BIT=1; else bt="x86_64"; fi
   - wget "https://repo.continuum.io/miniconda/${mc}-latest-${os}-${bt}.sh" -O miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - . $HOME/miniconda/etc/profile.d/conda.sh

--- a/.travis.yml
+++ b/.travis.yml
@@ -38,7 +38,8 @@ install:
   # version is the same.
   - if [[ "$PY" == "2.7" ]]; then mc=Miniconda2; else mc=Miniconda3; fi
   - if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then os=Linux; else os=MacOSX; fi
-  - wget https://repo.continuum.io/miniconda/${mc}-latest-${os}-x86_64.sh -O miniconda.sh
+  - if [[ "${BITS}" = "32" ]]; then bt="x86"; else bt="x86_64"; fi
+  - wget "https://repo.continuum.io/miniconda/${mc}-latest-${os}-${bt}.sh" -O miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - . $HOME/miniconda/etc/profile.d/conda.sh
   - hash -r
@@ -46,7 +47,6 @@ install:
   - conda update -q conda
   # Useful for debugging any issues with conda
   - conda info -a
-  - if [ "${BITS}" = "32" ]; then export CONDA_FORCE_32BIT=1; fi
   - conda create -q -n test-env python=$PY
   - conda activate test-env
   # Install kdb+

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,22 +6,22 @@ matrix:
       env:
         - BITS=64
         - PY=3.6
-        - NUMPY=1.14.3
+        - NUMPY=1.16.2
     - os: linux
       env:
         - BITS=64
         - PY=2.7
-        - NUMPY=1.14.3
+        - NUMPY=1.16.2
     - os: linux
       env:
         - BITS=32
         - PY=3.6
-        - NUMPY=1.14.3
+        - NUMPY=1.16.2
     - os: linux
       env:
         - BITS=32
         - PY=2.7
-        - NUMPY=1.14.3
+        - NUMPY=1.16.2
     - os: osx
       env:
         - BITS=64
@@ -32,14 +32,15 @@ matrix:
         - PY=2.7
   allow_failures:
     - os: osx
-    - env: BITS=32
+    - os: linux
+      env: BITS=32
 
 install:
   # We do this conditionally because it saves us some downloading if the
   # version is the same.
   - if [[ "$PY" == "2.7" ]]; then mc=Miniconda2; else mc=Miniconda3; fi
   - if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then os=Linux; else os=MacOSX; fi
-  - if [[ "${BITS}" = "32" ]]; then bt="x86"; export CONDA_FORCE_32BIT=1; else bt="x86_64"; fi
+  - if [[ "${BITS}" = "32" ]]; then bt="x86"; else bt="x86_64"; fi
   - wget "https://repo.continuum.io/miniconda/${mc}-latest-${os}-${bt}.sh" -O miniconda.sh
   - bash miniconda.sh -b -p $HOME/miniconda
   - . $HOME/miniconda/etc/profile.d/conda.sh

--- a/.travis.yml
+++ b/.travis.yml
@@ -33,7 +33,9 @@ matrix:
   allow_failures:
     - os: osx
     - os: linux
-      env: BITS=32
+      env: BITS=32 PY=2.7 NUMPY=1.16.4
+    - os: linux
+      env: BITS=32 PY=3.6 NUMPY=1.16.4
 
 install:
   # We do this conditionally because it saves us some downloading if the

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,22 +6,22 @@ matrix:
       env:
         - BITS=64
         - PY=3.6
-        - NUMPY=1.16.2
+        - NUMPY=1.16.4
     - os: linux
       env:
         - BITS=64
         - PY=2.7
-        - NUMPY=1.16.2
+        - NUMPY=1.16.4
     - os: linux
       env:
         - BITS=32
         - PY=3.6
-        - NUMPY=1.16.2
+        - NUMPY=1.16.4
     - os: linux
       env:
         - BITS=32
         - PY=2.7
-        - NUMPY=1.16.2
+        - NUMPY=1.16.4
     - os: osx
       env:
         - BITS=64

--- a/src/pyq/tests/test_com.py
+++ b/src/pyq/tests/test_com.py
@@ -24,7 +24,4 @@ def test_closed_connection(kdb_server):
     with pytest.raises(kerr) as info:
         kdb_server(cmd)
     msg = info.value.args[0]
-    if WIN:
-        assert msg.startswith('rcv.')
-    else:
-        assert msg == 'close'
+    assert msg.startswith('rcv.' if WIN else 'close')

